.eslintrc.js

```diff
@@ -1,34 +1,77 @@
 module.exports = {
-  parser: '@typescript-eslint/parser',
-  parserOptions: {
-    project: 'tsconfig.json',
-    sourceType: 'module',
-  },
-  plugins: ['@typescript-eslint/eslint-plugin'],
-  extends: [
-    'plugin:@typescript-eslint/eslint-recommended',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:prettier/recommended',
-    'prettier',
-  ],
   root: true,
   env: {
     node: true,
-    jest: true,
+    es2021: true,
   },
-  ignorePatterns: ['.eslintrc.js'],
-  rules: {
-    '@typescript-eslint/interface-name-prefix': 'off',
-    '@typescript-eslint/explicit-function-return-type': 'off',
-    '@typescript-eslint/explicit-module-boundary-types': 'off',
-    '@typescript-eslint/no-explicit-any': 'off',
-    'prettier/prettier': [
-      'error',
-      {
-        'singleQuote': true,
-        'semi': true,
-        'trailingComma': 'all'
+
+  parserOptions: { sourceType: 'module' },
+
+  overrides: [
+    {
+      files: ['.eslintrc.js'],
+      extends: ['plugin:prettier/recommended'],
```

Comment:
Here we are using prettier/recommended for this specific file **.eslintrc.js**, however it was formatted with eslint.

So it complains about the single quote used in this file instead of the double quote which seems to be the default for prettier:
![error](https://user-images.githubusercontent.com/36277587/188163507-30f1b8e2-0199-4cf5-8164-b71c9f2aec6d.png)